plugins/org.jkiss.dbeaver.ext.cubrid/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,17 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: DBeaver Cubrid Support
+Bundle-SymbolicName: org.jkiss.dbeaver.ext.cubrid;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Bundle-Release-Date: 20230123
+Require-Bundle: org.eclipse.core.runtime,
+ org.jkiss.dbeaver.model;visibility:=reexport,
+ org.jkiss.dbeaver.model.sql;visibility:=reexport,
+ com.google.guava;resolution:=optional,
+ org.jkiss.dbeaver.ext.generic
+Bundle-ActivationPolicy: lazy
+Bundle-RequiredExecutionEnvironment: JavaSE-17
+Bundle-Vendor: JKISS
+Import-Package: org.jkiss.dbeaver.tasks.nativetool,

```

Comment:
Any specific reason to have here these two imports?

Reply:
It may be my mistake to import those two without intention. I will remove it.